R/install_MACS.R

```diff
@@ -0,0 +1,40 @@
+#' install MACS python package
+#'
+#' @param envname Virtual environment to install the MACS python
+#'     module
+#' @param method Use conda or virtualenv to install MACS.
+#' @return install MACS python module
+#' @importFrom reticulate virtualenv_create virtualenv_install
+#'     virtualenv_list use_virtualenv conda_create conda_install
+#'     use_condaenv
+#' @export
+
+install_MACS <- function(envname = "MACS", method = "conda") {
+    stopifnot(is.character(envname) && length(envname) == 1)
+    is_windows <- identical(.Platform$OS.type, "windows")
+    is_osx <- Sys.info()["sysname"] == "Darwin"
+    is_linux <- identical(tolower(Sys.info()[["sysname"]]), "linux")
+    if (!is_windows && !is_osx && !is_linux) {
```

Comment:
I don't think MACS can be installed in windows through conda.

Reply:
OK. I will remove windows.

Reply:
zzz.R is a script to run when loading the package. For now, it only checks whether macs is installed.